5.findBugCommit.js

```diff
@@ -24,7 +24,20 @@
 //  }  
 
 const identifyFirstBadCommit = (commits) => {
-  
+  let left = 0;
+  let right = commits.length - 1;
+  while (left < right) {
+    let mid = Math.floor((left + right) / 2)
+      if(commits[mid]["status"] === "bad") {
+        right = mid
+      } else {
+        left = mid + 1
+      }
+  }
+  return {
+    commit: commits[timestamp],
+    commits: commits[status]
+  }
```

Comment:
No such thing as timestamp and status. You want to use your right variable. Also the problem is asking for the index.